Purchases/SubscriberAttributes/SubscriberAttributesManager.swift

```diff
@@ -32,111 +32,88 @@ class SubscriberAttributesManager {
     }
 
     func setAttributes(_ attributes: [String: String], appUserID: String) {
-        logAttributionMethodCalled(functionName: #function)
+        Logger.debug(Strings.attribution.setting_attributes(attributes: Array(attributes.keys)))
         for (key, value) in attributes {
             setAttribute(key: key, value: value, appUserID: appUserID)
         }
     }
 
     func setEmail(_ maybeEmail: String?, appUserID: String) {
-        logAttributionMethodCalled(functionName: #function)
-        setAttribute(key: SpecialSubscriberAttributes.email, value: maybeEmail, appUserID: appUserID)
+        setSpecialAttribute(.email, value: maybeEmail, appUserID: appUserID)
```

Comment:
every time i look at these i get a little confused what "special" means...they aren't really special, they're just already "created"... we do have a ticket for more refactoring of this code though, so not sure if a rename makes sense at this point anyway.

Reply:
I guess ["reserved"](https://docs.revenuecat.com/docs/subscriber-attributes#reserved-attributes) might be more appropriate

Reply:
i can add that potential rename to the ticket?

Reply:
oh, I already did it 53962f6